.pre-commit-config.yaml

```diff
@@ -1,16 +0,0 @@
-repos:
```

Comment:
which file does pre-commit configs now?

Reply:
The configuration of the auto gen'd file is in `pre-commit.nix`.